vue2-vuetify/src/complex/ArrayControlRenderer.vue

```diff
@@ -110,7 +114,7 @@
                         <v-icon class="notranslate">mdi-arrow-up</v-icon>
                       </v-btn>
                     </template>
-                    Move Up
+                    {{ t('array.btn.moveUp.tooltip', 'Move Up') }}
```

Comment:
We just merged a PR on the main repository which provides these translations: https://github.com/eclipsesource/jsonforms/pull/2129
We'll release on the main repository next week, then we can update this PR to consume these translations.

Reply:
@sdirix  I have changed most of the translations but for the up/down tooltip we are missing the properties - we have only the up/down for the aria label - or should I use that for now ?

Reply:
good point. let's keep it like this for now